toolkit/tools/internal/pkggraph/pkggraph.go

```diff
@@ -1056,8 +1053,8 @@ func (g *PkgGraph) AddMetaNode(from []*PkgNode, to []*PkgNode) (metaNode *PkgNod
 			for _, n := range to {
 				toNames = fmt.Sprintf("%s %s", toNames, n.FriendlyName())
 			}
-			logger.Log.Errorf("Couldn't add meta node from [%s] to [%s]", fromNames, toNames)
-			logger.Log.Panicf("Adding meta node failed.")
+			err := fmt.Errorf("failed to add meta node from (%s) to (%s)", fromNames, toNames)
+			logger.Log.Panicf("Adding meta node failed:\n%v\n%v", err, r)
```

Comment:
Why bother creating the `err` object? Why not just add everything to the `logger.Log.Panicf()`?